### ISSUE TITLE
How to pass arguments to docker-run

### ISSUE THREAD

**Issue by lgaticaq** [ryanramage/docker-build-run-push#1]
Is possible to specify ports and environment variables in "docker-run"

``` javascript
{
  "name": "test-app",
  "version": "0.0.1",
  "main": "app/server",
  "scripts": {
    "start": "babel-node app/server",
    "nodemon": "nodemon --exec npm start",
    "test": "mocha --compilers js:babel/register",
    "docker-build": "./node_modules/docker-build-run-push/docker-build",
    "docker-run": "./node_modules/docker-build-run-push/docker-run -p 3000:3000 --env-file .env"
  },
  "engines": {
    "iojs": "1.0.x"
  },
  "dependencies": {
    "docker-build-run-push": "^1.2.1",
    "dotenv": "^1.2.0",
    "express": "^4.13.3"
  },
  "devDependencies": {
    "babel": "^5.8.23",
    "babel-core": "^5.8.23",
    "chai": "^3.2.0",
    "mocha": "^2.3.2",
    "nodemon": "^1.4.1"
  },
  "eslintConfig": {
    "env": {
      "node": true,
      "mocha": true,
      "es6": true
    },
    "ecmaFeatures": {
      "arrowFunctions": true,
      "blockBindings": true,
      "defaultParams": true,
      "destructuring": true,
      "forOf": true,
      "modules": true,
      "templateStrings": true,
      "jsx": true,
      "classes": true
    },
    "rules": {
      "semi": [2, "never"],
      "curly": [2, "multi-line"],
      "no-underscore-dangle": 0
    }
  }
}
```

**Comment by ryanramage** [comment]
Hmm, I am not sure about having those in the package.json

I usually pass args to docker when I run the command on the shell, eg

```
> npm run docker-run -- -p 3000:3000 --env-file .env
```

This uses the npm trick of everything after the -- being passed. You could try adding that in the package.json and see if it works, something like

```
"docker-run": "./node_modules/docker-build-run-push/docker-run -- -p 3000:3000 --env-file .env"
```

**Comment by ryanramage** [comment]
Sorry my mistake, the -- gets executed in the spawned docker container... for example I run

```
npm run docker-run -- node ./bin/cli.js
```

I will have to think how to do this...

**Comment by ryanramage** [comment]
Ok, I have tried something here. I have added a way to split the arguments, so they can be passed to both docker options and then the docker image. See the new readme for examples:

https://github.com/ryanramage/docker-build-run-push#docker-run-arguments

I had to choose the '+' as there are so few options for a shell character with overloaded meanings! Let me know if this works for you.

**Comment by ryanramage** [comment]
Published in 4.0.0. in commit https://github.com/ryanramage/docker-build-run-push/commit/c3546c2301d30bbb3f1621681a9bd3c38804d746
